rustfmt-config/src/config_type.rs

```diff
@@ -365,12 +373,50 @@ macro_rules! create_config {
                     self.set().width_heuristics(WidthHeuristics::null());
                 }
             }
+
+            fn set_license_template(&mut self) {
+                let license_template_path = self.license_template_path();
+                let mut license_template_file = match File::open(&license_template_path) {
+                    Ok(file) => file,
+                    Err(e) => {
+                        eprintln!("Warning: unable to open license template file {:?}: {}",
+                                  license_template_path, e);
+                        return;
+                    }
+                };
+                let mut license_template_str = String::new();
+                match license_template_file.read_to_string(&mut license_template_str) {
+                    Ok(_) => (),
```

Comment:
you could use `if let Err(e) = ...` rather than `match` here.

Reply:
Thanks, of course, done :)